File: jobs/2026-06-01__10-59-51/shared_team_memory_py__yQqhWCb/artifacts/user/myproject/test_search3.py
import os
from alchemyst_ai import AlchemystAI

client = AlchemystAI()
try:
    res = client.v1.context.search(
        minimum_similarity_threshold=0.0,
        similarity_threshold=1.0,
        query="test",
        metadata="true",
        body_metadata={"session_id": "test_session_xyz"}
    )
    for c in res.contexts:
        print(c.metadata)
except Exception as e:
    print("ERROR:", e)

File: jobs/2026-07-04__06-18-47/alchemyst_context_arithmetic_ts__YAuB6VY/artifacts/user/myproject/test_search3.js
const AlchemystAI = require('@alchemystai/sdk').default;
async function go() {
  const c = new AlchemystAI({ apiKey: process.env.ALCHEMYST_AI_API_KEY });
  try {
    const r = await c.v1.context.search({
      query: 'engineering product API release version notes',
      similarity_threshold: 0.1,
      minimum_similarity_threshold: 0.1,
      scope: 'internal',
      mode: 'standard',
      body_metadata: {
        groupName: ['eng', 'v1']
      },
      metadata: 'true'
    });
    const ctxs = (r && r.contexts) || [];
    console.log('count=' + ctxs.length);
    for (const c of ctxs) {
      console.log('  -', (c.content||'').slice(0, 80));
    }
  } catch (e) {
    console.error('ERR', e.status, e.message);
    console.error('BODY', e.error);
  }
}
go();
